config/samples/groupsnapshotclass-v1beta1.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: groupsnapshot.storage.openshift.io/v1beta1
+kind: VolumeGroupSnapshotClass
+metadata:
+  name: csi-hostpath-groupsnapclass
+driver: hostpath.csi.k8s.io
```

Comment:
use cephcsi examples in this sample folder

Reply:
Done